qa-tests-backend/src/test/groovy/ImageSignatureVerificationTest.groovy

```diff
@@ -213,6 +225,14 @@ nzTe7BpOmVwmqLkIefEJe5L4PSXtp2KFLZqGO/kY5A==
             .setCommand(["sleep", "600"])
             .setNamespace(SIGNATURE_TESTING_NAMESPACE)
 
+    // Deployment holding an image signed by keyless Cosign using public Sigstore instances.
+    static final private Deployment KEYLESS_SIGSTORE_DEPLOYMENT = new Deployment()
+            .setName("keyless-sigstore")
+            .setImage("quay.io/rhacs-eng/qa-signatures:keyless-sigstore@" + KEYLESS_SIGSTORE_IMAGE_DIGEST)
```

Comment:
nit: `"` make it GString with some features like https://groovy-lang.org/syntax.html#_string_interpolation
```suggestion
            .setImage("quay.io/rhacs-eng/qa-signatures:keyless-sigstore@$KEYLESS_SIGSTORE_IMAGE_DIGEST")
```

```
.setImage('quay.io/rhacs-eng/qa-signatures:keyless-sigstore@' + KEYLESS_SIGSTORE_IMAGE_DIGEST)
```